cli/src/subcommand.rs

```diff
@@ -102,10 +102,16 @@ fn get_github_private_key(
     }
 }
 
+fn concat(a: &[String], b: &[String]) -> Vec<String> {
+    // from: https://stackoverflow.com/a/53476705/3720597
+    [a, b].concat()
```

Comment:
for some reason using `[&paths, &base_file_paths)].concat()` inline doesn't work and the first thing I tired: `&paths + &base_file_paths` also didn't work